lib/matplotlib/tri/triangulation.py

```diff
@@ -126,7 +126,7 @@ def get_cpp_triangulation(self):
 
     def get_masked_triangles(self):
         """
-        Return an array of triangles that are not masked.
+        Return an array of triangles taking the mask into account.
```

Comment:
I think this is less clear than before. How is the mask taken into account? That could also mean a masked array.

Reply:
I agree it is more vague in the sense that it is not clear what it means. But considering the related docs-string (last sentence):
https://github.com/matplotlib/matplotlib/blob/0a4866f9869043bdec45eebd58b1f955b2197e98/lib/matplotlib/tri/triangulation.py#L25-L28

one may argue that at least now it is consistent, so clearer from that aspect (and not potentially contradicting each other). But I am open to suggestions how to improve both.

Reply:
Maybe simply
```suggestion
        Return an array of all non-masked triangles.
```

and
```
         up the triangle, ordered in an anticlockwise manner. If you don't want 
         the triangles filter out by the *mask*, use `get_masked_triangles` instead.

Reply:
Sorry, I do not fully get that they are saying the same thing. "non-masked" (and "not masked") can to me also mean "remaining after the mask has been applied".

What about
```
Return an array of all triangles, ignoring the mask.
```
(If that now is the correct behaviour, the name `get_masked_triangles` seems to indicate the opposite to me... And either way, a method named like that should probably not have a doc-string claiming that you get the non-masked triangles...)

Reply:
The "masked" wording is slightly confusing here. Conceptually, we use it in the same way as numpy masked arrays: The mask is a Boolean flag per element, and when True that element is undefined. We are usually only interested in the valid elements, i.e. the ones where the mask flag is False. We quite sloppily use "masked triangles" or "triangles after application of the mask" for these valid elements, but technically, we keep the elements that are *not* masked (I.e. where the mask is False).

In that sense, the name `get_masked_triangles` is really misleading. The problem is that we cannot change that easily as it is public API. Clarifying the docstring is the least we can do. Maybe we can also check the wording of numpy masked array documentation, since they should have the same issue.

Reply:
I think that the currently suggested wording is quite good. It is also consistent with the wording in the constructor.

Your description was more or less what I envisioned. `get_masked_triangles` sort of makes sense if the description does not include "not masked".